cvat-canvas/src/scss/canvas.scss

```diff
@@ -48,12 +48,15 @@ polyline.cvat_shape_drawing_opacity {
     font-weight: bold;
     fill: white;
     cursor: default;
-    font-family: Calibri, Candara, Segoe, 'Segoe UI', Optima, Arial, sans-serif;
-    text-shadow: 0 0 4px black;
+    filter: drop-shadow(1px 1px 1px black) drop-shadow(-1px -1px 1px black);
     user-select: none;
     pointer-events: none;
 }
 
+.cvat_canvas_text_dimensions {
+    fill: cyan;
```

Comment:
Why cyan and not white? It feels we have 2 styles for the same thing now

Reply:
To display different things in different colors
When everything is white, it is more difficult to read